e2e/cases/stylus/environment/src/index.js

```diff
@@ -0,0 +1,4 @@
+import './a.styl';
+import style from './b.module.styl';
+
+console.log(style);
```

Comment:
It is a good idea to console print during E2E tests?

Reply:
To prevent the bundler from removing unused `style' object.

This is client code, so the `console.log` won't affect the E2E logs.